### PR TITLE
URNs, Links, Templates, Schema, Defaults

### PR DESCRIPTION
**Pull request by anneschuth**
In this PR I introduce a few changes to the instrument registry format by changing the example iama.yaml.

- Move to urns (instead of IDs)
- Move to links being lists of objects (instead of strings)
- Move to system card templates to make it clear why answers even appear
- Use 'open_question' instead of 'open question' as a task type
- Remove measures folder (leftover)
- Formatted the yaml file
- Introduced a proper schema and schema validation
- Adds all defaults from MinBZK/python-project-template

### CODE REVIEW

**Review by GuusjeJuijn — COMMENTED** [review]
Should we also adjust the json schema according to this new link structure?

**Comment by GuusjeJuijn** [comment]
Looks good! 👍

**Comment by anneschuth** [comment]
@GuusjeJuijn I've added a proper schema and schema validation as well

File: schema.json [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "urn": {
+          "type": "string",
+          "pattern": "^urn:nl:aivt:ir:[0-9]+\\.[0-9]+"
```
Comment (anneschuth) [inline schema.json:25]: We should probably pick/register an appropriate urn.

File: instruments/iama.yaml [hunk 2]
```diff
@@ -1,657 +1,654 @@
 systemcard_path: .assessments[]               # append iama.yaml to 'assessments' list in system card 
-schema_version: v_1.0.0
+schema_version: 1.1.0
 
 instrument:
   name: "Impact Assessment Mensenrechten en Algoritmes (IAMA)"
   description: "Het IAMA helpt om de risico's voor mensenrechten bij het gebruik van algoritmen in kaart te brengen en maatregelen te nemen om deze aan te pakken."
-  version: 1.0
+  urn: "urn:nl:aivt:ir:1.0"
   language: "Nederlands"
```
Comment (ChristopherSpelt) [inline instruments/iama.yaml:8]: Make this "Dutch", or maybe better an Enum with values from ISO 639 language codes.

**Review by ChristopherSpelt — COMMENTED** [review]
Just one small comment, but looks very good!

File: instruments/iama.yaml (hunk 2 above)
Comment (anneschuth) [reply instruments/iama.yaml:8]: Thanks, yes, totally missed this

**Pull request merged**